assets/stylesheets/common/discourse-reactions.scss

```diff
@@ -130,60 +130,103 @@ html.discourse-reactions-no-select {
   }
 
   .counters {
-    border-top: 1px solid var(--primary-low);
     display: grid;
-    grid-template-columns: repeat(6, 1fr);
+    grid-template-columns: repeat(1, 1fr);
     grid-column-gap: 0.5em;
     grid-row-gap: 0.5em;
     padding: 0.5em;
     margin-bottom: auto;
   }
+}
+
+.discourse-reactions-state-panel-reaction-users {
+  .user-container {
+    box-shadow: shadow("card");
+    background-color: var(--secondary);
+    display: flex;
+    flex-direction: column;
+    padding: 0.5em;
+    z-index: 1;
+  }
 
   .users {
+    cursor: pointer;
+    align-items: center;
     display: grid;
-    grid-template-columns: repeat(8, max-content);
+    grid-template-columns: repeat(12, 20px);
     grid-column-gap: 0.5em;
     grid-row-gap: 0.5em;
     padding: 0.5em;
     margin-bottom: auto;
+  }
 
-    .trigger-user-card {
-      display: flex;
-      align-items: center;
-      justify-content: center;
+  .header-buttons {
+    display: inline-flex;
+    justify-content: space-between;
+  }
+
+  .count {
+    color: var(--primary);
+    font-size: $font-up-1;
+    margin-left: 0.5em;
+  }
+
+  .hide-users {
```

Comment:
why all the !important here ?

Reply:
Because It was using its specific CSS colors and was not accessing our CSS

Reply:
Yes I know what's the effect of important is :) I will have look after merging and see if I can't change this, maybe specificity in core is too high.